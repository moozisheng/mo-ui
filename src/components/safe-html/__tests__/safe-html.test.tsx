
import { render, cleanup, fireEvent, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import React from 'react';
import SafeHtml from '../index';

afterEach(cleanup);

test('renders TagList with base use', () => {
  const { container } = render( <SafeHtml html={'<b>我是使用b标签加粗的HTML内容</b>'} />);
  expect(container).toMatchSnapshot();
});





