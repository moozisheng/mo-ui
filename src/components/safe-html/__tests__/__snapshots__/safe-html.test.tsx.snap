// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TagList with base use 1`] = `
<div>
  <span
    class="safe-html"
  >
    <b>
      我是使用b标签加粗的HTML内容
    </b>
  </span>
</div>
`;
